### DISCUSSION TITLE
Web UI

### DISCUSSION THREAD

**Discussion by hypery2k** [micromdm/micromdm#1026] [Ideas]
Is an web app for micromdm planned?

**Comment by jessepeterson** [comment]
No, not as a part of this project at this time.

That said I'd take a look at the [Ecosystem page](https://github.com/micromdm/micromdm/wiki/Ecosystem) especially at DEPTool as well as [Xavier](https://github.com/jefferyabbott/Xavier) for 3rd-party web GUIs that work with MicroMDM.
